.github/workflows/trigger_branch_automation.yaml

```diff
@@ -0,0 +1,18 @@
+name: Trigger Branch Creation Automation in Kubeflow-ci repository
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
Since we now use `main` instead of `master` in all our repos, should we delete this?
```suggestion

```